lib/lightning/collections/collection.ex

```diff
@@ -42,4 +44,31 @@ defmodule Lightning.Collections.Collection do
       message: "A collection with this name already exists"
     )
   end
+
+  defp validate_changeset(changeset) do
+    changeset
+    |> validate_format(:name, ~r/^[a-z0-9]+([\-_.][a-z0-9]+)*$/,
+      message: "Collection name must be URL safe"
```

Comment:
It would be helpful for the users to know the exact rule. Something like only digits, letters and the separating symbols '-', '_' or '.'

Reply:
Good point. However, their is a very very low chance for users to get this error because we do a conversion "automagically" for them. 

They use the `form_changeset/2` down below which only deals with `raw_name`.